zerver/tests/fixtures/saml/idp.crt

```diff
@@ -0,0 +1,17 @@
+MIIDojCCAoqgAwIBAgIGAW0svbVqMA0GCSqGSIb3DQEBCwUAMIGRMQswCQYDVQQGEwJVUzETMBEG
```

Comment:
Just to confirm, I assume these are test cert values that aren't useful for any non-test authentication?

Reply:
Yeah, random keys generated on my computer. The IdP public key was the public key from my test okta setup, which I used because the samlresponse test fixture has signatures matching it, but since due to the mocking trick we're able to skip their validation, I swapped it out for another random key I generated.